File: Solutions/293/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main
{

    public static void main(String[] args)
    {
        Scanner scanner=new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        for(int i = a; i <= b ;i ++)
        {
            if(is_prime(i) == 1)
                System.out.println(i);
        }
    }

    private static int is_prime(int num)
    {
        if(num == 1)
            return 0;
        for(int i = 2; i < num; i ++)
        {
            if(num % i == 0)
                return 0;
        }
        return 1;
    }
}

File: Solutions/293/c++/program.cpp
// Copy from Quera

#include <iostream>
using namespace std;

int is_prime(int num)
{
    if (num == 1)
        return 0;
    for (int i = 2; i < num; i++)
    {
        if (num % i == 0)
            return 0;
    }
    return 1;
}

int main()
{
    int a, b;
    cin >> a >> b;
    for (int i = a; i <= b; i++)
    {
        if (is_prime(i) == 1)
            cout << i << endl;
    }
    return 0;
}